g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/translation/BundlesGenerator.java

```diff
@@ -62,7 +61,7 @@ public void handleRemoteRusult(String remoteRusult) {
                                     + TranslationUtil.genernateJsonLocalizedFileName(tLocale),
                             component, tLocale, (JSONObject) bundleObj.get(tLocale));
                 }
-            } catch (ParseException e) {
+            } catch (RuntimeException e) {
                 e.printStackTrace();
```

Comment:
Please check all of this kind of issues.

Reply:
this model can't use log4j